File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatMaterail = []

    def __init__(self, license, driver,typeCarAccepted, seatMaterail):
        super(UberBlack,self).__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatMaterail = seatMaterail

File: Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String, ArrayList<String,Integer>> typeCarAccepted;
    ArrayList<String> seatMaterial;

    public UberBlack(String license, Account driver,
    Map<String, ArrayList<String,Integer>> typeCarAccepted,
    ArrayList<String> seatMaterial) {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    }
}